flask/flask-launcher/build.gradle

```diff
@@ -0,0 +1,69 @@
+plugins {
+    id 'java-library'
+}
+
+evaluationDependsOn(":flask:flask-common")
```

Comment:
I can't see anything in this module which might need `evaluationDependsOn`. Referring to other projects in a `dependencies` block is normal for Gradle.